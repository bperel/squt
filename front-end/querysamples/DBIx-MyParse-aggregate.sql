SELECT COUNT(DISTINCT count_field), non_count_field
FROM table_name
GROUP BY non_count_field
