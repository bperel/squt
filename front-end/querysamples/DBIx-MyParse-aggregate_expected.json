{
  "Tables" : {
    "table_name" : {
      "table_name" : {
        "GROUP" : {
          "non_count_field" : "ASC"
        },
        "OUTPUT" : {
          "non_count_field" : {
            "-1" : "non_count_field"
          },
          "count_field" : {
            "0" : "count_field"
          }
        }
      }
    }
  },
  "Warning" : {
    "No alias" : {
      "COUNT_DISTINCT_FUNC" : "SELECT"
    }
  },
  "Functions" : {
    "0" : {
      "group" : "1",
      "alias" : "0",
      "name" : "COUNT_DISTINCT_FUNC",
      "to" : "OUTPUT"
    }
  }
}